pkg/plugin/plugin_test.go

```diff
@@ -337,6 +338,79 @@ var _ = Describe("Plugin - test CNI command flows", func() {
 				Expect(updatedPluginConf.MAC).To(BeIdenticalTo(testValidMAC3))
 			})
 		})
+
+		Context("Update DeviceInfo", func() {
+			var (
+				tmpFile string
+			)
+
+			BeforeEach(func() {
+				f, err := os.CreateTemp("", "accbr-deviceinfo")
+				Expect(err).NotTo(HaveOccurred())
+				tmpFile = f.Name()
+			})
+
+			AfterEach(func() {
+				Expect(os.Remove(tmpFile)).NotTo(HaveOccurred())
+			})
+
+			It("succeed", func() {
+				cmdCtx := &cmdContext{pluginConf: &localtypes.PluginConf{}}
+				cmdCtx.pluginConf.Representor = "eth3"
+				cmdCtx.pluginConf.RuntimeConfig.CNIDeviceInfoFile = tmpFile
+
+				By("unsupported spec version, should update the version")
```

Comment:
could you split to separate test cases like the bad flows below ?

usually `By` statements are used to provide user for information of complex flow in a single test.

Reply:
fixed